examples/differenceChart.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width" />
+    <link href="../build/nv.d3.css" rel="stylesheet" type="text/css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.2/d3.min.js" charset="utf-8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.19.2/moment.js"></script>
+    <script src="../build/nv.d3.js?dsf"></script>
+
+    <style>
+        text {
+            font: 12px sans-serif;
+        }
+        svg {
+            display: block;
+        }
+        html, body, svg {
+            margin: 0px;
+            padding: 0px;
+            height: 100%;
+            width: 100%;
+        }
+    </style>
+</head>
+<body class='with-3d-shadow with-transitions'>
+<button onclick="expandLegend();">Expand/Contract Legend</button>
+<script>
+  var expandLegend = function() {
+      var exp = chart.multiChart.legend.expanded();
+      chart.multiChart.legend.align(false);
+      chart.update();
+  }
+</script>
+<svg id="chart2"></svg>
+<div style="height: 1000px; background-color: red;">
+
+</div>
+<script>
+  let actual, predicted, data, chart;
+
+  async function readTextFile(file) {
+    return new Promise((resolve) => {
+      var rawFile = new XMLHttpRequest();
+      rawFile.overrideMimeType('application/json');
+      rawFile.open('GET', file, true);
+      rawFile.onreadystatechange = function() {
+        if (rawFile.readyState === 4 && rawFile.status == '200') {
+          resolve(rawFile.responseText);
+        }
+      }
+      rawFile.send(null);
+    });
+  }
+
+  const keyForActualLessThanPredicted = 'Savings (kWh)';
+  const keyForActualGreaterThanPredicted = 'Excess Usage (kWh)';
+
+  async function loadData() {
+    actual = await readTextFile('./actual.json');
+    predicted = await readTextFile('./predicted.json');
+    const actualData = JSON.parse(actual);
+    const predictedData = JSON.parse(predicted);
+
+    data = [{
+      key: 'Actual usage (kWh)',
+      type: 'actual',
+      values: actualData[0].values,
+      color: 'black'
+    }, {
+      key: 'Predicted usage (kWh)',
+      type: 'expected',
+      values: predictedData.map(d => {
+        return {
+          recorded_at: d.falling_edge,
+          value: d.predicted
+        };
+      }),
+      color: 'blue'
+    }];
+  }
+
+  async function main() {
+    const colors = d3.scale.category20();
+
+    await loadData();
```

Comment:
I've tried to keep IE11 support still, which doesn't support async/await.  I know that's annoying, but it's still supported by a lot of products including the one I work on for my real job.

Reply:
I guess since that's just the example it's okay though.

Reply:
I've converted it to ES5 :)